Dockerfile

```diff
@@ -4,17 +4,22 @@ WORKDIR /app
 
 COPY . /app
 
-RUN apt-get update && \
-    apt-get install -y curl && \
-    curl -sL https://deb.nodesource.com/setup_21.x | bash - && \
-    apt-get install -y nodejs && \
-    npm install -g yarn && \
-    cd src/main/resources/frontend && \
-    yarn && \
-    yarn build
+# RUN apt-get update && \
+#    apt-get install -y curl && \
+#    curl -sL https://deb.nodesource.com/setup_21.x | bash - && \
+#    apt-get install -y nodejs && \
+#    npm install -g yarn && \
+#    cd src/main/resources/frontend && \
+#    yarn && \
+#    yarn build
```

Comment:
I think this should stay enabled by default so that Docker remains an option for production deployments

Reply:
I forgot to uncomment that, that's on me.